editor/editor_log.cpp

```diff
@@ -254,8 +255,13 @@ void EditorLog::add_message(const String &p_msg, MessageType p_type) {
 	}
 }
 
-void EditorLog::set_tool_button(Button *p_tool_button) {
-	tool_button = p_tool_button;
+void EditorLog::_set_dock_tab_icon(Ref<Texture2D> p_icon) {
+	// This is the sole reason to import TabContainer here.
```

Comment:
You mean...?
```suggestion
	// This is the sole reason to include "tab_container.h" here.
```
Or similar